platform/shared/src/lib.rs

```diff
@@ -158,6 +162,39 @@ impl LoggerHolder {
     holder.shutdown(false);
     drop(holder);
   }
+
+  /// Logs an out-of-the-box app launch TTI log event. The method should be called only once.
+  /// Consecutive calls have not effect.
+  pub fn log_app_launch_tti(&self, duration: time::Duration) {
+    self.app_launch_tti_log.call_once(|| {
+      let duration_ms = duration.as_seconds_f64() * 1_000f64;
+      if duration_ms < 0.0 {
+        log::warn!(
+          "dropping app launch TTI log: reported TTI is negative: {}",
+          duration_ms
+        );
+        return;
+      }
+
+      let fields = vec![AnnotatedLogField {
+        field: LogField {
+          key: "_duration_ms".into(),
+          value: duration_ms.to_string().into(),
+        },
+        kind: LogFieldKind::Ootb,
+      }];
+
+      self.log(
+        log_level::INFO,
+        LogType::Lifecycle,
+        "AppLaunchTTI".into(),
```

Comment:
It could be argued that this value should be just a field in the existing app launch event, but I see how exposing it in our public api would make that complicated, so I guess this will have to do for now